.github/dependabot.yml

```diff
@@ -118,6 +118,9 @@ updates:
       - dependency-name: com.amazonaws:aws-lambda-java-events
       # Apache Commons
       - dependency-name: commons-io:commons-io
+      # Micrometer
+      - dependency-name: io.micrometer:micrometer-bom
+      - dependency-name: io.micrometer:micrometer-registry-stackdriver
```

Comment:
Why add this dependency here? Isn't already in the `micrometer-bom`?

Reply:
As explained to me by Guillaume, because we include stackdriver specifically in the application bom (we have to because we need to exclude transitive dependencies), we need to include it here, too.